docker-compose.ci.yml

```diff
@@ -72,3 +72,19 @@ services:
       - indy-pool
     tty: true
     privileged: true
+
+  tails-server:
+    image: kivaprotocol/tails-server:latest
+    ports:
+      - 6543:6543
+    networks:
+      agency-network:
+        ipv4_address: 10.0.0.3
```

Comment:
Does this need to be a hard coded ip address? It makes it easier if it's not hard coded

Reply:
Would you recommend with just 0.0.0.0?

Reply:
You can just leave it blank - ie no ipv4_address line